functions.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+
+function pokemonCardContainer(array $allPokemon): string {
+    $pokemonNames = "";
+    foreach ($allPokemon as $pokemon) {
+        $pokemonNames .=
+            '<section class="card-stats">' .
+            "<h2> Name: " . $pokemon['name'] . "</h2>".
+            '<img src="' . $pokemon['image'] . '" alt="' . $pokemon['name'] . '">' .
+            "<p> Cost: " .$pokemon['cost'] . "</p>" .
+            "<p> Rarity: ". $pokemon['rarity'] . "<p/>" . "</section>";
```

Comment:
this html is different than what the test expects so the unit test for this function fails